cpp/src/parquet/types.h

```diff
@@ -479,6 +479,9 @@ struct Encoding {
   };
 };
 
+// Exposed data encodings.
```

Comment:
please expand the comment to indicate where this is used and how this is different then actual encodings in the parquet format.